DataSet.rst

```diff
@@ -1,414 +1,451 @@
+        </pbds:Filters>
+        <pbds:DataSetMetadata>
+            <pbbase:TotalLength>3000</pbbase:TotalLength>
+            <pbbase:NumRecords>300</pbbase:NumRecords>
+        </pbds:DataSetMetadata>
+    </pbds:SubreadSet>
+
+
+    <pbds:SubreadSet xmlns="http://pacificbiosciences.com/PacBioDataModel.xsd">
+        <pbbase:ExternalResources>
+            <pbbase:ExternalResource MetaType="SubreadFile.SubreadBamFile"
                                ResourceId="file:/mnt/path/to/subreads1.bam"/>
-        </ExternalResources>
-        <Filters>
-            <Filter>
-                <Parameter Name="rq" Value=">0.75"/>
-            </Filter>
-        </Filters>    
-        <DataSetMetadata>
-            <TotalLength>2000</TotalLength>
-            <NumRecords>200</NumRecords>
-        </DataSetMetadata>
-    </SubreadSet>
-
-
-Consolidating DataSets
-++++++++++++++++++++++
+        </pbbase:ExternalResources>
+        <pbds:Filters>
+            <pbds:Filter>
+                <pbbase:Properties>
+                    <pbbase:Property Name="rq" Operator="gt" Value="0.75"/>
+                </pbbase:Properties>
+            </pbds:Filter>
+        </pbds:Filters>
+        <pbds:DataSetMetadata>
+            <pbbase:TotalLength>2000</pbbase:TotalLength>
+            <pbbase:NumRecords>200</pbbase:NumRecords>
+        </pbds:DataSetMetadata>
+    </pbds:SubreadSet>
+
+
+Tools that merge multiple DataSets together can optionally store the original input DataSets
+in the DataSets tag of the output DataSet. In this way the original DataSets are maintained
+as "subdatasets" of the new DataSet. Merging tools should adhere to the following best practices\:
+
+- Subdatasets should be created for each input DataSet when two or more files with no subdatasets are merged or used to create a new DataSet.
+- If in the process of merging a new DataSet is created, for example, wrapping a "naked" BAM file, a subdataset should be created for those inputs.
+- In no case should a subdataset be present if it is identical to the containing DataSet.
+- Subdatasets should be preserved when two datasets with subdatasets are merged together.
+- If both have subdatasets, the lists of subdatasets should be concatenated.
+- If one has no subdatasets, a subdataset should be created for that input and added to the list of subdatasets from the other input file.
+- Subdatasets should be preserved during minor manipulations: Adding a filter, changing path absoluteness, adding or removing indices, references etc.
+- Subdatasets should be removed during substantial transformations: alignment, producing an AlignmentSet from a SubreadSet.
+- Subdatasets should be preserved during consolidation or splitting (except when splitting by subdataset).
+
+
+Consolidating
++++++++++++++
 
 Consolidating (aka Resolving) a DataSet means creating an explicit
 representation in the appropriate format with all filters applied. Here
 is consolidated version of the SubreadSet above::
```

Comment:
This sounds a bit abstract.  Maybe add that "In practice, this means building a single BAM file containing all the records implied by the dataset's input and filter directives."